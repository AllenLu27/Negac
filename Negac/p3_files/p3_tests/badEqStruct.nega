struct S {
	bool field;
};

void main(){
	struct S a;
	struct S b;
	if (a != b){
	}
}
