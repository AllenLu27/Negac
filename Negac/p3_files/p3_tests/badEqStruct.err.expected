FATAL [8,8]: Invalid equality operation
Semantic analysis failed
